api/policies/hasValidMoveBody.js

```diff
@@ -27,6 +27,20 @@ module.exports = function (req, res, next) {
       }
     }
       return next();
+
+    case MoveType.SCUTTLE: {
+      if (!cardId && !targetId) {
+        return res.badRequest({ message: 'Must specify a card' });
+      }
+
+      const invalidCardId = [cardId, targetId].find(id => !DeckIds.includes(id));
```

Comment:
I think this would read clearer with .some
```suggestion
      const invalidCardId = [cardId, targetId].some(id => !DeckIds.includes(id));
```